packages/cli/src/cli/config/action.imagery.config.ts

```diff
@@ -120,13 +128,12 @@ export class CommandImageryConfig extends CommandLineAction {
     if (commit) {
       logger.info({ path }, 'ImageryConfig:UploadConfig');
       const configJson = provider.toJson();
-      const output = fsa.join(path, `basemaps-config-${configJson.hash}.json.gz`);
-      await fsa.writeJson(output, configJson);
-      const configPath = base58.encode(Buffer.from(output));
-      logger.info(
-        { path: output, url: `https://basemaps.linz.govt.nz/?config=${configPath}&tileMatrix=NZTM2000Quad${location}` },
-        'ImageryConfig:Done',
-      );
+      const outputPath = fsa.join(path, `basemaps-config-${configJson.hash}.json.gz`);
+      await fsa.writeJson(outputPath, configJson);
+      const configPath = base58.encode(Buffer.from(outputPath));
+      const url = `https://basemaps.linz.govt.nz/?config=${configPath}&i=${name}&tileMatrix=NZTM2000Quad${location}`;
+      logger.info({ path: output, url }, 'ImageryConfig:Done');
+      if (output != null) fsa.write(output, url);
```

Comment:
you need to `await` this

Reply:
Good pick. Fixed.

Reply:
is there anyway in argo to just set a env var or do we always have to use files for outputs?

Reply:
We should investigate this later. Currently what I see is just parameter output and artifacts output.